dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -774,17 +774,17 @@ def prepare_provider_packages(
     perform_environment_checks()
     fix_ownership_using_docker()
     cleanup_python_generated_files()
-    temp_provider_packages = None
+    packages_list_as_tuple: tuple[str, ...] = ()
     if package_list and len(package_list):
         get_console().print(f"\n[info]Populating provider list from PACKAGE_LIST env as {package_list}")
         # Override provider_packages with values from PACKAGE_LIST
-        temp_provider_packages = tuple(package_list.split(","))
-    if provider_packages and package_list:
+        packages_list_as_tuple = tuple(package_list.split(","))
+    if provider_packages and packages_list_as_tuple:
         get_console().print(
             f"[warning]Both package arguments and --package-list / PACKAGE_LIST passed. "
-            f"Overriding to {temp_provider_packages}"
+            f"Overriding to {packages_list_as_tuple}"
         )
-    provider_packages = temp_provider_packages or ()
+    provider_packages = packages_list_as_tuple or provider_packages
```

Comment:
This is where problem was - we did not pass the original provider packages if package_list was empty, we passed empty list instead.